src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="rate__block">
+      <div class="stars stars--0">
+        <div class="stars_img"></div>
+        <div class="stars_img"></div>
+        <div class="stars_img"></div>
+        <div class="stars_img"></div>
+        <div class="stars_img"></div>
+      </div>
+      <div class="stars stars--1">
```

Comment:
checklist point 1 (but do not leave empty lines if the tag contains only 1 line)
```suggestion
      </div>

      <div class="stars stars--1">
```